### ISSUE TITLE
cymbol: load symbols from header files

### ISSUE THREAD

**Issue by disconnect3d** [pwndbg/pwndbg#1973]
The `cymbol ...` command should support adding symbols from a header file. That should make it easier to use it in scripts etc.

**Comment by Tanmay-rajk** [comment]
Is this issue still open? I would like to work on this.

**Comment by disconnect3d** [comment]
@Ravenspar Yes it is. You can always check the functionality and see its not there ;). Feel free to send a PR.

**Comment by 0xRavenspar** [comment]
Hey, I'm new to Open Source. Kindly forgive me if I made any mistakes while sending this PR.

I've sent a pr #2661 to implement this feature. 

Here's a summary of the changes:
1. Added the `-f` flag to the `cymbol` command.
2. Implemented tests for the new functionality.
3. Updated the documentation to include the usage of the `-f` flag.

Can you please review it and guide me on any changes I should make.

**Comment by disconnect3d** [comment]
@0xRavenspar Hi, see comments in PR :)
